.github/labels.yaml

```diff
@@ -0,0 +1,28 @@
+- color: '30ABB9'
+  description: This PR will be backported to v1.0
+  name: backport-v1.0
+- color: 'BCF611'
+  description: A good issue for first-time contributors
+  name: good first issue
+- color: '9E1957'
+  description: Breaking change
+  name: semver:major
+- color: 'FBCA04'
+  description: Backwards-compatible change
+  name: semver:minor
+- color: '6E7624'
+  description: No API change
+  name: semver:patch
+- color: 'D73A4A'
+  description: Do not merge
+  name: hold
+- color: 'F9D0C4'
+  description: Additional information requested
+  name: needinfo
+- color: 'FEF2C0'
+  description: This PR lacks tests before it can be merged
+  name: just-needs-tests
```

Comment:
Actually I am not sure this label was a good idea in Gophercloud

Reply:
Sure, I can remove this label if we think we're not going to use it.